packages/xrpl/src/sugar/submit.ts

```diff
@@ -187,29 +122,28 @@ async function waitForFinalTransactionOutcome<
     )
   }
 
-  const txResponse = await client
-    .request<TxRequest, TxResponse<T>>({
-      command: 'tx',
-      transaction: txHash,
-    })
-    .catch(async (error) => {
-      // error is of an unknown type and hence we assert type to extract the value we need.
-      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions,@typescript-eslint/no-unsafe-member-access -- ^
-      const message = error?.data?.error as string
-      if (message === 'txnNotFound') {
-        return waitForFinalTransactionOutcome<T>(
-          client,
-          txHash,
-          lastLedger,
-          submissionResult,
-        )
-      }
-      throw new Error(
-        `${message} \n Preliminary result: ${submissionResult}.\nFull error details: ${String(
-          error,
-        )}`,
-      )
-    })
+  const txResponse = await client.request<TxRequest, TxResponse<T>>({
+    command: 'tx',
+    transaction: txHash,
+  })
+  // .catch(async (error) => {
+  //   // error is of an unknown type and hence we assert type to extract the value we need.
+  //   // eslint-disable-next-line @typescript-eslint/consistent-type-assertions,@typescript-eslint/no-unsafe-member-access -- ^
+  //   const message = error?.data?.error as string
+  //   if (message === 'txnNotFound') {
+  //     return waitForFinalTransactionOutcome<T>(
+  //       client,
+  //       txHash,
+  //       lastLedger,
+  //       submissionResult,
+  //     )
+  //   }
+  //   throw new Error(
+  //     `${message} \n Preliminary result: ${submissionResult}.\nFull error details: ${String(
+  //       error,
+  //     )}`,
+  //   )
+  // })
```

Comment:
Why was this error handling commented out? (I believe this was originally added due to a race condition which could occur where the transaction wasn't quite validated yet which was a very intermittent error)

Reply:
Added back